finalfrontier-utils/src/lib.rs

```diff
@@ -1,3 +1,6 @@
+#[macro_use]
+extern crate clap;
```

Comment:
This shouldn't be necessary. In `util`, you should just be able to do `use clap::crate_version`.